server/odc-migrate/src/main/resources/init-config/init/regulation-rule-metadata.yaml

```diff
@@ -1422,4 +1422,29 @@
       type: INTEGER
       componentType: INPUT_NUMBER
       defaultValues:
-        - !!int 1000
\ No newline at end of file
+        - !!int 1000
+- id: 60
+  name: ${com.oceanbase.odc.builtin-resource.regulation.rule.sql-check.estimate-sql-affected-rows-failed.name}
+  description: ${com.oceanbase.odc.builtin-resource.regulation.rule.sql-check.estimate-sql-affected-rows-failed.description}
+  type: SQL_CHECK
+  builtIn: 1
+  labels:
+    - label: SUB_TYPE
+      value: DML
+    - label: SUB_TYPE
+      value: UPDATE
+    - label: SUB_TYPE
+      value: DELETE
+    - label: SUB_TYPE
+      value: INSERT
+    - label: SUPPORTED_DIALECT_TYPE
+      value: OB_MYSQL
+    - label: SUPPORTED_DIALECT_TYPE
+      value: MYSQL
+  propertyMetadatas:
+    - name: ${com.oceanbase.odc.builtin-resource.regulation.rule.sql-check.estimate-sql-affected-rows-failed.unable-to-estimate-sql-affected-count}
+      description: ${com.oceanbase.odc.builtin-resource.regulation.rule.sql-check.estimate-sql-affected-rows-failed.unable-to-estimate-sql-affected-count.description}
+      type: INTEGER
```

Comment:
what's this input for?

Reply:
actually, `propertyMeatadatas`  it's not needed here.